### PR TITLE
[PR] Fix 'Eventos' page

### LINKED ISSUE #41: Events/services section review
https://mosteirodelandim.surge.sh/services

This section needs a final review before going live given that we now have a strong lean towards weddings as these are the bread and butter that keeps the place afloat (pretty much).

### PR DESCRIPTION
**Pull request by LuchoTurtle**
closes #41 

Missing:

- [x] bottom images
- [x] replace text with price of a cover
- [x] send screenshots to Inês of the photos that **weren't** changed so they can get credited.

### CODE REVIEW

File: src/assets/images/events/carousel/4.jpg
Comment (nelsonic) [inline src/assets/images/events/carousel/4.jpg:1]: recognise this one. 😜

File: src/pages/events.js [hunk 1]
```diff
@@ -169,14 +163,15 @@ function EventsPage({data}) {
...
+              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel1} alt='photocredits to Conta-me Como Foi' title='photocredits to Conta-me Como Foi'/>
+
+              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel2} alt='photocredits to João Makes Photos' title='photocredits to João Makes Photos'/>
+
+              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel3} alt='photocredits to Saiotes' title='photocredits to Saiotes'/>
+              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel4} alt='photocredits to Madalena Abreu' title='photocredits to Madalena Abreu'/>
+              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel5} alt='photocredits to Pedro Marcelino' title='photocredits to Pedro Marcelino'/>
+              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel6} alt='photocredits to Pedro Marcelino' title='photocredits to Pedro Marcelino'/>
+              
               <BackgroundImage Tag="div" className="carousel-image" fluid={carousel9}/>
```
Comment (LuchoTurtle) [inline src/pages/events.js:175]: Will probably leave the last picture as it's not worth it to wait for one single image to be chosen.

**Review by LuchoTurtle — COMMENTED** [review]
Approving this PR because it's "good enough" as is.

File: src/assets/images/events/carousel/6.jpg
Comment (nelsonic) [inline src/assets/images/events/carousel/6.jpg:1]: Much better (higher) resolution. Does it still download fast on a slower (mobile) connection? 💭

Comment (LuchoTurtle) [reply src/assets/images/events/carousel/6.jpg:1]: Yes, `gastby-image` creates thumbnails and lower resolution pictures for loading effects and mobile devices.

Comment (nelsonic) [reply src/assets/images/events/carousel/6.jpg:1]: Noice. 👌

**Pull request merged**